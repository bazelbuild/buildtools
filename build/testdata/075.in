manifest = [
    # buildifier: table
    ("", "", ""),
    ("ab", "bcbcbcbcbcbcbcbcbcbcbc", "cd"),
    ("aaaaaaaaaaaaaaaaaaaa", "b", "c"),
]

