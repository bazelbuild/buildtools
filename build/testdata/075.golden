load("@bar.bzl", "x", "y")
load("@foo//:foo.bzl", "x", "y")
load("//:baa.bzl", "x", "y")
load("//:bac.bzl", "x", "y")
load(":bab.bzl", "x", "y")
